src/stories/Forms/TextArea/index.tsx

```diff
@@ -2,25 +2,16 @@ import React from 'react';
 import styled from 'styled-components';
 import rem from '../../../shared/pxToRem';
 import { GlobalInputProps } from '../../../shared/types';
+import { StyledInput } from '../Input';
 
-interface WrapperProps {
-  hasError?: boolean;
-  height: number;
-}
+const Wrapper = styled.div`
+  flex: 1;
+`;
 
-const Wrapper = styled.div<WrapperProps>`
-  textarea {
-    display: block;
-    box-sizing: border-box;
-    width: 100%;
-    height: ${(p) => rem(p.height)};
-    padding: var(--spacing-16);
-    border: ${(p) => (p.hasError ? 'var(--border-error)' : 'var(--border-primary)')};
-    border-radius: var(--input-radius);
-    font-size: var(--input-font-size);
-    font-family: var(--font-family-primary);
-    resize: vertical;
-  }
+const StyledTextArea = styled(StyledInput)<{ hasError?: boolean; height: number }>`
```

Comment:
Prop has error is not used anymore

Reply:
It is for inputs though, its handled in the wrapper and passed down to input fields as Boolean. 
See [InputWrapper Line 89](https://github.com/Huddly/huddly-storybook/blob/main/src/stories/Forms/InputWrapper/index.tsx#89s)